section2/exercises/ex30.rb

```diff
@@ -0,0 +1,58 @@
+# assigns values to three variabls
```

Comment:
This conditionals practice looks solid. I like that your syntax is really consistent. Paying attention to all of those small details makes your code really easy to read!